tests/omni_lock_rust/tests/misc.rs

```diff
@@ -1086,6 +1146,194 @@ impl Default for MultisigTestConfig {
     }
 }
 
+#[derive(Clone, Copy)]
+pub enum BitcoinSignVType {
+    P2PKHUncompressed,
+    P2PKHCompressed,
+    SegwitP2SH,
+    SegwitBech32,
+}
+impl Default for BitcoinSignVType {
+    fn default() -> Self {
+        Self::P2PKHCompressed
+    }
+}
+
+#[derive(Default)]
+pub struct BitcoinConfig {
+    pub sign_vtype: BitcoinSignVType,
+    pub pubkey_err: bool,
+}
+
+impl BitcoinConfig {
+    pub fn get_pubkey_hash(&self, pubkey: &Pubkey) -> [u8; 20] {
+        if self.pubkey_err {
+            let mut r = [0u8; 20];
+            thread_rng().fill_bytes(&mut r);
+            return r;
+        }
+        match self.sign_vtype {
+            BitcoinSignVType::P2PKHUncompressed => {
+                let mut pk_data = Vec::<u8>::new();
+                pk_data.resize(65, 0);
+                pk_data[0] = 4;
```

Comment:
define a const value like this:
https://github.com/bitcoin-core/secp256k1/blob/d373bf6d08c82ac5496bf8103698c9f54d8d99d2/include/secp256k1.h#L221